src/main/resources/db/migration/20220803__premises_tables.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE ap_area (
```

Comment:
Can we add a time to the migration name here too, just thinking if we generate more than one migration a day, it could get confusing. Might be good to have a generate-migration script to make this easier. Something like:

```
timestamp=`date "+%Y%m%d%H%M%S"`
filename=`echo "$1" | awk '{print tolower($0)}' | tr ' ' '_'`

touch "src/main/resources/db/migration/$timestamp_$filename"
```

Which can then be run with `script/geneate-migration migration name goes here`

Reply:
Amazing 👍